config/betterquesting/DefaultQuests/Quests/Tier8UV-AAAAAAAAAAAAAAAAAAAALQ==/IPutFusioninYour-AAAAAAAAAAAAAAAAAAAEsg==.json

```diff
@@ -29,7 +29,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "To make Oganesson for your Void Miners, you\u0027ll need to make Californium, to make that, you\u0027ll need Beryllium and Plutonium 239, all in the Reactor.\n\nSeriously consider replicating it after the first cell."
+      "desc:8": "To make Oganesson, you\u0027ll need to make Californium. To make that, you\u0027ll need Beryllium and Plutonium 239, all in the Fusion Reactor. After the two fusion chains, you\u0027ll have the best noble gas in the pack, which is used in several places.\n\nIn this tier, you can send it to Void Miners, to increase their ore production speed, or EBFs, to make them run faster, although the EBF option is too weak right now (changes coming soon). Later on, Oganesson will be used in other places, like the optical circuit line, or to produce Metastable Oganesson, which is needed in much larger quantities later on.\n\nSeriously consider replicating it after the first cell."
```

Comment:
are we still recommending replication? how do the numbers look with your change. are we not planning to remove that?

Reply:
I'll change the text to instead reflect that replication is on the way out, and the fusion recipe is better now.